<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-URKGA8H" name="testCases.Facility" tests="21" failures="0" timestamp="02 May 2018 07:33:36 GMT" time="1190.566" errors="1">
  <testcase name="Tc_Patterns_02" time="60.118" classname="testCases.Facility"/>
  <testcase name="Tc_Dep_03_04" time="68.360" classname="testCases.Facility"/>
  <testcase name="Tc_Patterns_05" time="64.561" classname="testCases.Facility"/>
  <testcase name="Tc_Phy_05" time="76.926" classname="testCases.Facility"/>
  <testcase name="Tc_Patterns_06" time="17.457" classname="testCases.Facility">
    <error type="java.lang.NumberFormatException" message="For input string: &quot;1525246259665&quot;">
      <![CDATA[java.lang.NumberFormatException: For input string: "1525246259665"
	at java.lang.NumberFormatException.forInputString(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at testCases.Facility.Tc_Patterns_06(Facility.java:763)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- Tc_Patterns_06 -->
  <testcase name="Tc_Dep_06" time="62.412" classname="testCases.Facility"/>
  <testcase name="Tc_Manufacturers_02" time="29.744" classname="testCases.Facility"/>
  <testcase name="Tc_Fac_01" time="88.964" classname="testCases.Facility"/>
  <testcase name="Tc_Manufacturers_01" time="20.601" classname="testCases.Facility"/>
  <testcase name="Tc_Fac_02" time="35.098" classname="testCases.Facility"/>
  <testcase name="Tc_Patterns_01_03" time="52.643" classname="testCases.Facility"/>
  <testcase name="Tc_Dep_07" time="68.082" classname="testCases.Facility"/>
  <testcase name="Tc_Phy_01" time="42.668" classname="testCases.Facility"/>
  <testcase name="Tc_Fac_03" time="32.033" classname="testCases.Facility"/>
  <testcase name="Tc_Fac_05_Dep_02" time="60.551" classname="testCases.Facility"/>
  <testcase name="Tc_Fac_04" time="35.781" classname="testCases.Facility"/>
  <testcase name="Tc_GL_Code_01_02" time="74.965" classname="testCases.Facility"/>
  <testcase name="Tc_Phy_01_02_03" time="104.796" classname="testCases.Facility"/>
  <testcase name="Tc_Phy_04" time="80.675" classname="testCases.Facility"/>
  <testcase name="Tc_GL_Code_04_05" time="93.066" classname="testCases.Facility"/>
  <testcase name="Tc_ManageApprovalFlow_01" time="21.065" classname="testCases.Facility"/>
</testsuite> <!-- testCases.Facility -->
